README.md

```diff
@@ -237,7 +237,7 @@ terminal.write("Some text\n Some text on new line");
 ### Input Reading
 This module enables you to read user input events.
 
-Good documentation could be found on the following places: [docs](https://docs.rs/crossterm_input/), [book](https://timonpost.github.io/crossterm/docs/input.html), [examples](https://github.com/TimonPost/crossterm/tree/master/examples/key_events.rs)
+Good documentation could be found on the following places: [docs](https://docs.rs/crossterm_input/), [book](https://timonpost.github.io/crossterm/docs/input.html), [examples](https://github.com/crossterm-rs/crossterm/tree/master/examples/key_events.rs)
```

Comment:
I'd suggest to generate pages, setup them now and change these links as well. Pages are the only thing that are not redirected.